Library/Homebrew/cmd/install.rb

```diff
@@ -227,6 +227,12 @@ def check_development_tools
       out = checks.send(check)
       opoo out unless out.nil?
     end
+    if OS.mac? && MacOS.prerelease?
```

Comment:
Pull this logic into `all_development_tools_checks` instead to avoid duplication and using `OS.mac?` here.